doc-generator/src/main/scala/codacy/pylint/Main.scala

```diff
@@ -21,18 +22,18 @@ object Main {
       for {
         file <- File.temporaryFile()
         _ = file.write(html)
-        res = Seq("pandoc", "-f", "html", "-t", "markdown", file.pathAsString).!!
+        res = Seq("pandoc", "-f", "html", "-t", "commonmark", file.pathAsString).!!
       } yield res
     result.get()
   }
 
   val docsPath = "../docs"
 
   val version: String = {
-    val file = File(docsPath) / "patterns.json"
-    val patterns = file.contentAsString
-    val json = ujson.read(patterns)
-    json("version").str
+    val file = File("../requirements.txt")
+    file.lines.collectFirst {
```

Comment:
Codacy found an issue: [Usage of get on optional type.](https://app.codacy.com/gh/codacy/codacy-pylint/pullRequest?prid=6163400)